### ISSUE TITLE
Consider rendering the change file in a different way

### ISSUE THREAD

**Issue by ematipico** [knope-dev/knope#930]
At the moment, each change file contains a summary that will be used as header (`###` or `####`) when the changelog is computed. 

I fear that using too many headers could make the changelog too divided, especially for those projects that might contain **a lot** of changes (fixes and features).

I would like to know if you're open to the idea of rendering the change files in a different way.

### No header

I like how [`changesets`](https://github.com/changesets/changesets) renders the changelog. Each change file is just an item inside a bullet list.

Since `scopes` aren't supported with change files, rendering in a different way would give more freedom to the users. For example, given these two files:

```md
---
pizza: patch
---

## Fixed size of the pizza

There was something wrong with the size.
```

```md
---
pizza: patch
---

## Fixed name of the pizza
```

It would render in a changelog like this:

```md

## Bug fixes

- **Fixed size of the pizza**
    There was something wrong with the size.
- **Fixed name of the pizza**

```

### Group headers

Another alternative would be to group **headers** from different change files into one entry. For example:

```md
---
restaurant: patch
---

## Menu

Fixed a typo in the desserts
```

```md
---
restaurant: patch
---

## Menu

Fixed a typo in the starters.
```

It would render in a changelog like this:

```md

## Bug fixes

### Menu

- Fixed a typo in the desserts
- Fixed a typo in the starters.
```

## Applicability

Theoretically, both suggestions should work with the existing format of the change files. 

If not, maybe it's possible to evaluate using an option inside `knope.toml`, and also prefixing the change files with a `v2_*.md`./

**Comment by dbanty** [comment]
Knope was originally using bullets for changes, but once we started supporting changesets, the generated changelogs became too hard to follow. Since you can put arbitrary Markdown for each change, you can put bullets and code blocks and other things that break the flow unless each change file has a header. There is still some Markdown you can put in changesets which will break the flow of the log (for example, headers), but it's much more flexible. In addition, headers usually result in links in their rendered form (like GitHub releases), so it's easy to point someone at a particular change when answering a question.

Perhaps, though, there's a way to go to a hybrid style, where complex changesets get their own little subsection, but simple changes are summarized as bullets in the top section? So conventional commits would always be bullets, and change files would be bullets as long as there was no content outside the header.

## Changes

- `feat: A feature`
- `fix: A quick fix`

```markdown
---
pizza: patch
---

## Fixed name of the pizza
```

```markdown
---
pizza: patch
---

## Fixed size of the pizza

- Big pizzas were too small
- Small pizzas were also too small

> Can never have enough pizza!
```

## Generated Changelog Section

### Features

- A feature

### Bug fixes

- A quick fix
- Fixed name of the pizza

#### Fixed size of the pizza

- Big pizzas were too small
- Small pizzas were also too small

> Can never have enough pizza!

**Comment by ematipico** [comment]
This kinda of approach would be amazing!
